src/Avalonia.Themes.Default/Accents/BaseLight.xaml

```diff
@@ -20,7 +21,7 @@
     <SolidColorBrush x:Key="ErrorBrush">Red</SolidColorBrush>
     <SolidColorBrush x:Key="ErrorBrushLight">#10ff0000</SolidColorBrush>
 
-    <sys:Double x:Key="ThemeBorderThickness">2</sys:Double>
+    <avalonia:Thickness x:Key="ThemeBorderThickness">2</avalonia:Thickness>
```

Comment:
`Thickness` should be picked up without a namespace. To do this you need to add an `XmlnsDefinition` attribute to the `Avalonia.Visuals` assembly with the plain `Avalonia` namespace. There are already a couple of these definitions: https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Visuals/Properties/AssemblyInfo.cs#L11